entries/p/pro.btcturk.com.json

```diff
@@ -0,0 +1,16 @@
+{
+  "BtcTurk Pro": {

```

Comment:
Is the authentication process different from the non-pro version of btcturk.com?

Reply:
I guess yes, i never use non-pro version of btcturk.com

Reply:
From what I can tell `pro.btcturk.com` and `btcturk.com` both redirect to `sso.btcturk.com` for the authentication part. Furthermore https://www.btcturk.com/yardim/guvenlik doesn't mention any difference between the pro and non-pro version in terms of authentication so I would recommend you rename the file and domain element to `btcturk.com`.